examples/adb-uc/stage_3_spn_deploys_uc/terraform.tfvars

```diff
@@ -1 +1 @@
-databricks_resource_id = "/subscriptions/3e551f82-c624-4a3e-b8f6-e050a96a7004/resourceGroups/testrg/providers/Microsoft.Databricks/workspaces/ws1"
+databricks_resource_id = "/subscriptions/3e551f82-c624-4a3e-b8f6-e050a96a7004/resourceGroups/hwang-demo-rg/providers/Microsoft.Databricks/workspaces/demo-adb-ws"
```

Comment:
Use `azure_databricks_workspace` data source instead to retrieve the workspace ID by workspace name  and resource group.

Reply:
thanks @alexott , this folder directly piece together the code blocks in tf guide - https://registry.terraform.io/providers/databricks/databricks/latest/docs/guides/unity-catalog-azure where the resource_id of workspace is fed as input, and if I use data source again it will introduce a loop.

Shall I restructure it and use TF to spin up a workspace to feed these attributes instead of relying on data?